android/src/main/java/com/it_nomads/fluttersecurestorage/ciphers/RSACipher18Implementation.java

```diff
@@ -137,16 +137,20 @@ private void createKeys(Context context) throws Exception {
                     .setEndDate(end.getTime())
                     .build();
         } else {
-            spec = new KeyGenParameterSpec.Builder(KEY_ALIAS, KeyProperties.PURPOSE_DECRYPT | KeyProperties.PURPOSE_ENCRYPT)
+            KeyGenParameterSpec.Builder builder = new KeyGenParameterSpec.Builder(KEY_ALIAS, KeyProperties.PURPOSE_DECRYPT | KeyProperties.PURPOSE_ENCRYPT)
                     .setCertificateSubject(new X500Principal("CN=" + KEY_ALIAS))
                     .setDigests(KeyProperties.DIGEST_SHA256)
                     .setBlockModes(KeyProperties.BLOCK_MODE_ECB)
                     .setEncryptionPaddings(KeyProperties.ENCRYPTION_PADDING_RSA_PKCS1)
                     .setCertificateSerialNumber(BigInteger.valueOf(1))
                     .setCertificateNotBefore(start.getTime())
-                    .setCertificateNotAfter(end.getTime())
-                    .setIsStrongBoxBacked(true)
-                    .build();
+                    .setCertificateNotAfter(end.getTime());
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
```

Comment:
@bbedward I added this as a fix for exception https://github.com/mogol/flutter_secure_storage/pull/40#issuecomment-460938184